src/components/Checkout/GuestUpsell.vue

```diff
@@ -9,7 +9,7 @@
 		<form id="guestUpsellForm" action="." @submit.prevent.stop="submit">
 			<kv-base-input
 				name="firstName"
-				class="data-hj-suppress"
+				class="data-hj-suppress tw-mb-2"
```

Comment:
The guest create account form was a little cramped, this looks better, see screenshot